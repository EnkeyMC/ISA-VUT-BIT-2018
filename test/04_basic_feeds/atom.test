NAME=Basic Atom parsing
ARGS=http://www.mocky.io/v2/5bd3027e3400000f00cfdfdb
RC=0
CHECK_OUTPUT=1
