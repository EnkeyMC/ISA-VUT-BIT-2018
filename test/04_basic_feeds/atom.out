*** xkcd.com ***
I'm a Car
Carnot Cycle
Barnard's Star
Tectonics Game
